Fix altitude bounds check (#81)

rtucker opened the pull request:
Permit altitudes up to 17999.99.  Move the limits to be with
the rest of the hardcoded values.

Fixes issue #81

kb1lqc committed:
Cleaned up some upper() statements and forced upper on configuration file callsign read.

kb1lqc committed:
Cleaned up telemetry to use a % SQL wildcard instead of a None type for callsign queries. Therefore, I also removed the check for None and simply always generate an SQL querying for either the supplied callsign or a % which has the same effect as not specifying a callsign but will always survive an upper() python conversion.

kb1lqc committed:
Updated APRS application to no longer use a login.ini file and instead calculate the APRS passcode automatically. Does this expose the algorith? Yes. Does that matter? No. It's super easy and already online. Hiding the algorithm serves no purpose. If APRS-IS doesn't like it then they need to move to a public/private key based system, it's 2017.

kb1lqc committed:
Updated Readme to account for passcode generaton

kb1lqc committed:
Check for generic REPLACEME as callsign to warn user if they forgot to update the callsign field in the configuration file. Also updated screenshot documentation image.

kb1lqc committed:
Changed socket exception checking from socket.error to ERRORIO in an effort to more cleanly and cross-platform detect socket errors and fail gracefully.

kb1lqc committed:
Merge branch 'master' into issue91

Pulling in updates from Master branch.

kb1lqc committed:
Created issue template file

kb1lqc committed:
Spelling error and formatting fix

rtucker committed:
Fix altitude bounds check (#81)

Permit altitudes up to 17999.99.  Move the limits to be with
the rest of the hardcoded values.

rtucker committed:
Fix altitude bounds check (#81)

Permit altitudes up to 17999.99.  Move the limits to be with
the rest of the hardcoded values.

rtucker committed:
fix typo in previous commit

(thought this was committed, argh)

rtucker committed:
fix typo in previous commit

(thought this was committed, argh)

rtucker committed:
Add links to github label filters

rtucker committed:
Spellcheck/typofixes

kb1lqc commented on Faraday_Proxy_Tools/FaradayIO/deviceconfig.py:
```diff
@@ -285,7 +285,7 @@ def update_gps(self, gps_boot_bitmask, latitude_str, latitude_dir_str, longitude
         else:
             print "ERROR: GPS string(s) too long OR NMEA DMM formatting incorrect"
-            print "ERROR: Altitude must be %g-%g" % (MIN_ALTITUDE_METERS, MAX_ALTITUDE_METERS)
+            print "ERROR: Altitude must be %g-%g" % (self.MIN_ALTITUDE_METERS, self.MAX_ALTITUDE_METERS)
```
Hey @rtucker thanks for the pull request! How about we use Python .format() instead of % for string variable and formatting? This would correspond to the rest of Faraday Software + is the preferred method as of Python 2.6. It's also the preferred method for Python 3.0 if we move towards that down the line.

kb1lqc reviewed (changes requested):
Simple python preference and maintainability change preferred but otherwise this looks good. Make the update and I'll pull it in!

kb1lqc committed:
Merge pull request #101 from rtucker/contrib-doc-fixup

CONTRIBUTING.md minor fixes

kb1lqc committed:
Merge pull request #104 from FaradayRF/issuetemplate

Implement Issue Template

kb1lqc committed:
Quick fix to allow Telemetry and APRS to work

Needed to add "%" as default for callsign in stations request. Pull #93 fixes this but that includes other updates that will slow it down. I Know this works and it's well tested.

kb1lqc committed:
Added newline between image and text

kb1lqc committed:
Spelling error

kb1lqc committed:
Merge branch 'master' into issue91

kb1lqd committed:
Merge pull request #93 from FaradayRF/issue91

Telemetry and APRS Callsign + Passcode Update

rtucker commented:
I agree, good call.  I'll fire up the laptop and fix this :-) -rt

rtucker committed:
Use format method instead of printf-style subs

rtucker committed:
Merge branch 'issue-81' of github.com:rtucker/Faraday-Software into issue-81

rtucker commented:
Ah fudge, got distracted and f'd up my branch.  Grumble...

Will open a new clean pull request.
